### PR TITLE
fota: change progress value to percentage downloaded

### PR DESCRIPTION
**Pull request by jayteemo**
changed the progress value to percentage downloaded to enable the front-end to correctly display the progress of the download.

### CODE REVIEW

**Comment by sigvartmh** [comment]
Added DNM need to be discussed with cloud first if we can stop our self from bloating the JSON even more.

**Comment by sigvartmh** [comment]
I guess I didn't explain myself too well.

Instead of adding any API changes just change
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/9bcea0487fe28674271960a41211f5157cb492f5/subsys/net/lib/fota_download/src/fota_download.c#L112
to
```
send_progress((offset*100 / file_size);
```

This way you don't have to store any stored_file_size in ` aws_fota.c`  and you won't have to send the total in the progress event.

**Comment by jayteemo** [comment]
> I guess I didn't explain myself too well.
> 
> Instead of adding any API changes just change
> https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/9bcea0487fe28674271960a41211f5157cb492f5/subsys/net/lib/fota_download/src/fota_download.c#L112
> 
> 
> to
> ```
> send_progress((offset*100 / file_size);
> ```
> 
> This way you don't have to store any stored_file_size in ` aws_fota.c` and you won't have to send the total in the progress event. You could probably change the name in statusDetails to progress or something else than offset when it is done.

my fault, your explanation was just fine 👍

File: subsys/net/lib/fota_download/src/fota_download.c [hunk 1]
```diff
@@ -108,8 +108,16 @@ static int download_client_callback(const struct download_client_evt *event)
...
+			}
+
+			if (file_size == 0) {
+				LOG_DBG("invalid file size: %d", file_size);
+				send_evt(FOTA_DOWNLOAD_EVT_ERROR);
+				return err;
 			}
-			send_progress(offset);
+
+			send_progress((offset*100)/file_size);
```
Comment (rlubos) [inline subsys/net/lib/fota_download/src/fota_download.c]: Please follow the coding standard and add spaces around operators.

**Comment by rlubos** [comment]
@sigvartmh Is the DNM still effective here?

**Comment by jayteemo** [comment]
Yes this PR can be closed.
It is handled by https://github.com/NordicPlayground/fw-nrfconnect-nrf/pull/1891

**Pull request closed without merging**